### DISCUSSION TITLE
What's the difference between "args", "argTypes", "parameters" in React?

### DISCUSSION THREAD

**Discussion by ryota-murakami** [storybookjs/storybook#17059] [Help] [answered]
I'm sorry pretty annoying question for here.  
But I only certain(80%) "argTypes" describe acceptable props list of component, 
although I Don't know how to use correctly other 2 field. 😭

**Comment by shilman** [comment, marked as answer]
- [argTypes & args](https://storybook.js.org/docs/react/writing-stories/args) - metadata about input to a component & the values of those inputs, respectively
- [parameters](https://storybook.js.org/docs/react/writing-stories/parameters) - other metadata about a story, typically to configure storybook features and addons

**Reply by ryota-murakami** [reply]
@shilman Thank you so mush!  
Each document covering enough ready to use, should skip document in the case of Storybook. 📚
